opm/input/eclipse/Schedule/ScheduleState.hpp

```diff
@@ -474,6 +475,8 @@ namespace Opm {
         map_member<int, VFPInjTable> vfpinj;
         map_member<std::string, Group> groups;
         map_member<std::string, Well> wells;
+        // constant flux aquifers
+        std::unordered_map<int, SingleAquiferFlux> aqufluxs;
```

Comment:
Did you try to make this into a `map_member<int, SingleAquiferFlux>` which is generally how `ScheduleState` manages its time-dependent objects?  If so, why did that not work?  If not, why not?

Reply:
Hei, with this, basically we know the exact `report step`  when this keyword is entered, similar to combining `ScheduleState` and `Events` together. 

It is easier to use this way, especially if we want to extend to cope the SCHEDULE part of AQUFETP and AQUCT, I thought this can be a good practice although these two keywords are not the focus yet.